.github/workflows/ci.yml

```diff
@@ -0,0 +1,74 @@
+name: CI-IoTBlock
+on:
+  push:
+    branches:
+      - build-from-ci
+
+jobs:
+  build:
+    name: Build IoT Block test app
+    runs-on: self-hosted
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v2
+        with:
+          path: "shard-v2-app-template"
+      - name: Get version
+        run: |
+          cd shard-v2-app-template
+          MAJOR=`cat common/global_consts.h | grep VITROIO_TEMPLATE_VERSION | cut -d " " -f 5 | sed -n 1p`
+          MINOR=`cat common/global_consts.h | grep VITROIO_TEMPLATE_VERSION | cut -d " " -f 5 | sed -n 2p`
+          PATCH=`cat common/global_consts.h | grep VITROIO_TEMPLATE_VERSION | cut -d " " -f 5 | sed -n 3p`
+          CURRENT_VERSION="$MAJOR.$MINOR.$PATCH"
+          echo "CURRENT_VERSION=${CURRENT_VERSION}" >> $GITHUB_ENV
+          echo "MAJOR=${MAJOR}" >> $GITHUB_ENV
+          echo "MINOR=${MINOR}" >> $GITHUB_ENV
+          echo "PATCH=${PATCH}" >> $GITHUB_ENV
+      - name: Apply patch
+        run: |
+          cd shard-v2-app-template
+          git apply patches/0001-docker_build_image.sh-change-docker-flag-to-suit-ci.patch
+      - name: Build app
+        run: |
+          cd shard-v2-app-template
+          chmod +x docker_build_image.sh
+          git submodule update --init
+          ./build_app.sh IoTBlock-test
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-region: eu-west-1
+      - name: Prepare files
+        run: |
+          echo $CURRENT_VERSION
+          sudo mv shard-v2-app-template/bin/app_a.bin app_a-${CURRENT_VERSION}.bin
+          sudo mv shard-v2-app-template/bin/app_b.bin app_b-${CURRENT_VERSION}.bin
+          sudo mv shard-v2-app-template/bin/image.bin image-${CURRENT_VERSION}.bin
+      - name: Copy files to AWS S3 bucket
+        run: |
+          aws s3 cp app_a-${CURRENT_VERSION}.bin s3://dev-vitro-shard-applications/shard-binaries-test/
```

Comment:
@esesyn01 You can remove that, we do not need app_a and app_b on the cloud. It will be OK if we upload only the final binary - `image-${CURRENT_VERSION}.bin`. Also, you can rename it to be more precise, like `iot-block-test-${CURRENT_VERSION}.bin`. Maybe in the future, we will store different applications than IoT-B test

Reply:
@kotylamichal Done